payload/payload.h

```diff
@@ -10,5 +10,10 @@ namespace sorbet::payload {
 void createInitialGlobalState(core::GlobalState &gs, const realmain::options::Options &options,
                               const std::unique_ptr<const OwnedKeyValueStore> &kvstore);
 
+// Create a copy of `gs` that has the same string storage, name table, and file table, but otherwise has its state set
+// as though it was initialized by `createInitialGlobalState`.
+std::unique_ptr<core::GlobalState> copyForSlowPath(const core::GlobalState &gs,
+                                                   const realmain::options::Options &options);
```

Comment:
Ditto for this being a little out of place, but given its reliance on the payload symbol table blob, I'm not sure there's a better place for it.